### PR TITLE
docs: Setup docs for `shiny.experimental`

### PR DESCRIPTION
**Pull request by schloerke**

### CODE REVIEW

File: shiny/experimental/ui/_tag.py [hunk 1]
```diff
@@ -40,20 +43,38 @@ def tag_remove_class(tag: TagT, x: str) -> TagT:
...
+        # If here, `attrs.class` must exist
         tag.attrs.pop("class")
         return tag
 
-    tag.attrs["class"] = " ".join(
-        [cls_val for cls_val in cls.split(" ") if cls_val != x]
-    )
+    # Otherwise, set the new class
+    tag.attrs["class"] = " ".join(new_cls)
     return tag
```
Comment (schloerke) [inline shiny/experimental/ui/_tag.py:72]: I adjusted this method to be take an array. 

Does it look okay?

File: shiny/experimental/ui/_tag.py [hunk 2]
```diff
@@ -16,22 +16,25 @@
...
-    tag.attrs["class"] = class_
+def tag_prepend_class(tag: TagT, *class_: str | None) -> TagT:
+    classes = (
+        *class_,
+        tag.attrs.get("class"),
+    )
+    classes = [c for c in classes if c is not None]
+    if len(classes) == 0:
+        return tag
+    tag.attrs["class"] = " ".join(classes)
```
Comment (schloerke) [inline shiny/experimental/ui/_tag.py:27]: I adjusted this method to take an array of classes.

Does it look okay?

File: shiny/experimental/ui/_fill.py [hunk 3]
```diff
@@ -254,85 +348,208 @@ def remove_all_fill(tag: TagT) -> TagT:
...
+
+    def as_fillable_container(
+        self: T,
+    ) -> T:
+        ...
+
+    def remove_all_fill(
+        self: T,
+    ) -> T:
+        ...
```
Comment (schloerke) [inline shiny/experimental/ui/_fill.py:527]: Is this approach of using two different class values a fair approach to make this section generic for classes and not just Tag objects?

**Review by schloerke — COMMENTED** [review]
These areas are questions I have about the code. 

Otherwise it is example apps and documentation.

**Review by wch — APPROVED** [review]

**Pull request merged**